docs/src/docs/asciidoc/gradle-plugin.adoc

```diff
@@ -836,57 +825,8 @@ include::../snippets/gradle/kotlin/build.gradle.kts[tags=include-metadata]
 
 For more advanced configurations you can declare the `org.graalvm.buildtools.gradle.tasks.CollectReachabilityMetadata` task and set the appropriate properties.
 
-[[pgo-support]]
```

Comment:
How to optimize with PGO is now part of end-to-end guide: https://github.com/graalvm/native-build-tools/blob/master/docs/src/docs/asciidoc/end-to-end-gradle-guide.adoc#gather-execution-profiles-and-build-optimized-images 
I don't think we should have it in 2 places. Plus, in Maven doc, there isn't any mention of PGO. The structure should be consistent, at lease close.

Reply:
Yes I agree, I think this should be documented in the main docs.

Reply:
What if we add a subtitle followed by a short paragraph that a user can build optimized images with PGO, and then  send them to  end-to-end guides?

Reply:
I don't know. Personally I'm not much interested in the end-to-end guide, so I see linking to it as an annoyance. I'd prefer to have all required information in the reference docs.